src/json_to_tsv.py

```diff
@@ -0,0 +1,308 @@
+# Reads harmonized JSON files and produces TSV files
+
+import csv
+import inflect
+import json
+import logging
+import sys
+import uuid
+from bento_mdf import MDF
+from pathlib import Path
+
+logger = logging.getLogger(__name__)
+
+# String values that we must assume
+FOREIGN_ID_SUFFIX = '_id'
+PARENT_NODE_ID = 'study_id'
+PARENT_NODE_NAME = 'study'
+PARENT_NODE_NAME_PLURAL = 'studies'
+PRIMARY_KEY_NAME = 'id'
+
+def main():
+    # Set up logging
+    logging.basicConfig(format='%(message)s', level=logging.INFO, stream=sys.stdout)
+
+    args = sys.argv[1:]
+
+    # Check CLI args
+    if len(args) < 3:
+        logger.error('Insufficient arguments. Provide 3 CLI arguments: data directory, model MDF path, and props MDF path.')
+        return
+
+    # Read CLI args
+    data_dir = Path(args[0]).resolve()
+    model_mdf_path = Path(args[1]).resolve()
+    props_mdf_path = Path(args[2]).resolve()
+
+    # Node names we expect from the JSON files
+    model = get_datamodel(model_mdf_path, props_mdf_path)
+    node_names = get_node_names(model)
+    dir_paths = get_data_subdirectories(data_dir)
+
+    logger.info(f'Found {len(dir_paths)} subdirectories\n')
+
+    # Look at all the files in the data directory, grouping files within a subdirectory into a single study
```

Comment:
Will move this loop to the `process_json_data()` function to help it run in Prefect, according to @qiongL910 's advice.